### PR TITLE
Feature/planning comparison

### PR DESCRIPTION
**Pull request by mugwump**
Restoring historical values for a given set of filtered work_packages.

### CODE REVIEW

**Comment by apfelfabrik** [comment]
Types, Statuses and Projects should be prefetched, as those trigger subsequent queries to the database, per work package that is included in the comparison.

**Comment by apfelfabrik** [comment]
Also, something triggers queries of the form

```
SELECT COUNT(*) FROM "work_packages" WHERE "work_packages"."parent_id" = 99
```

to the database, apparently also per work package that is included in the comparison.

**Comment by apfelfabrik** [comment]
```
Task #1880: Re-introduce at-time scope
Task #1881: Re-introduce project planning comparison in controller
Task #1994: Integrational tests for work packages at_time (API)
Task #2173: Adapt client-side to new server behavior
Task #1883: Extend at-time scope for status comparison
Task #1884: Make status values available over API
```

File: app/services/planning_comparison_service.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        wp.status  = statuses[wp.status_id]
+      end
+
+      work_packages
+
+    end
+
+    def self.resolve_reference_ids(work_packages)
+      # TODO faster ways to do this without stepping numerous times through the workpackages?!
+      # Or simply wait unti we finally throw out the redundant references out of the json/xml-rendering??!
```
Comment (hschink) [inline app/services/planning_comparison_service.rb]: Just a typo: `unti` should be `until` I guess...

File: features/timelines/timeline_comparison_view.feature.disabled [hunk 2]
```diff
@@ -0,0 +1,81 @@
+#-- copyright
```
Comment (hschink) [inline features/timelines/timeline_comparison_view.feature.disabled:1]: Why is this file deactivated?

Comment (apfelfabrik) [inline features/timelines/timeline_comparison_view.feature.disabled:1]: It is still in progress in `#1993`.

**Pull request merged**